backend/src/dtos/cars/autoria-plain-data.dto.ts

```diff
@@ -0,0 +1,8 @@
+export class AutoriaPlainDataDto {
+  name: string;
+  autoria_code: number;
+  constructor(data: { name: string; value: number }) {
+    this.name = data.name;
+    this.autoria_code = data.value;
+  }
+}
```

Comment:
To follow the exports style in other files, I would write the class like this:

```suggestion
class AutoriaPlainDataDto {
  name: string;
  autoria_code: number;
  constructor(data: { name: string; value: number }) {
    this.name = data.name;
    this.autoria_code = data.value;
  }
}

export { AutoriaPlainDataDto }
```